src/backend/drive/settings.py

```diff
@@ -333,7 +333,22 @@ class Base(Configuration):
 
     # Cache
     CACHES = {
-        "default": {"BACKEND": "django.core.cache.backends.locmem.LocMemCache"},
+        "default": {
```

Comment:
should we keep LocMemCache for local development and tests?

Reply:
For local dev we were already using `django_redis.cache.RedisCache` - it is a part of code I removed in the same commit below

Reply:
Anyway, I'm removing this commit it causes too much damages